osb-backup-manager/src/main/java/de/evoila/cf/backup/service/manager/BackupServiceManager.java

```diff
@@ -43,6 +43,7 @@ public BackupServiceManager(AbstractJobRepository abstractJobRepository,
         this.abstractJobRepository = abstractJobRepository;
         this.credentialService = credentialService;
         this.backupCleanupManager = backupCleanupManager;
+        log.debug("Debug activated");
```

Comment:
should this be removed?

Reply:
The log was added by me when we started looking for the problem. Thats why i removed it now. We can keep it if you find it useful

Reply:
You did not remove this log line. That's why I suggested to remove it.